build/crunchy-postgres-exporter/Dockerfile

```diff
@@ -1,32 +1,27 @@
-ARG BASEOS
-ARG BASEVER
-ARG PREFIX
-FROM ${PREFIX}/pgo-base:${BASEOS}-${BASEVER}
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
When we start supporting ubi9 this will have to use an argument to set which ubi image to base from.

Reply:
We have a couple of places where we hard-code ubi8 as the version. When we start supporting 9 we will have to find a solution for this, but I'm definitely kicking that can